Sources/ObservableStore/ObservableStore.swift

```diff
@@ -111,15 +143,40 @@ public struct Update<Model: ModelProtocol> {
 
     public init(
         state: Model,
-        fx: Fx<Model.Action> = Empty(completeImmediately: true)
-            .eraseToAnyPublisher(),
-        transaction: Transaction? = nil
+        fx: Fx<Model.Action>,
+        transaction: Transaction?
     ) {
         self.state = state
         self.fx = fx
         self.transaction = transaction
     }
 
+    public init(state: Model, animation: Animation? = nil) {
+        self.state = state
+        self.fx = Empty(completeImmediately: true).eraseToAnyPublisher()
+        self.transaction = Transaction(animation: animation)
+    }
+
+    public init(
+        state: Model,
+        fx: Fx<Model.Action>,
+        animation: Animation? = nil
+    ) {
+        self.state = state
+        self.fx = fx
+        self.transaction = Transaction(animation: animation)
+    }
+
+    public init(
+        state: Model,
+        future: Future<Model.Action, Never>,
```

Comment:
It may be worth dropping this initializer in favor of having user erase before passing in.

Rationale: you often want do additional work with publisher, such as merging with other publishers.

Reply:
Update: yeah, after test driving in app code, I think it's better to drop this constructor.